script.js

```diff
@@ -23,3 +23,239 @@ links.forEach((link) => {
+    cardButton1: 'https://loltolo-lesapiti.github.io/',
+    cardButton2: 'https://github.com/Loltolo-Lesapiti/loltolo-lesapiti.github.io',
+  },
+  {
+    class: 'flex reverse',
+    header2: 'Multi-Post Stories',
+    heading3: 'FACEBOOK',
+    list1: ['Full Stack Dev', '2015'],
+    paragraph: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    mobileImage: 'images/card2-image.png',
+    desktopImage: 'images/card2.png',
+    list2: ['html', 'css', 'javaScript'],
+    cardButton: 'See Project',
+    cardButton1: 'https://loltolo-lesapiti.github.io/',
+    cardButton2: 'https://github.com/Loltolo-Lesapiti/loltolo-lesapiti.github.io',
+  },
+  {
+    class: 'flex',
+    header2: 'Facebook 360',
+    heading3: 'FACEBOOK',
+    list1: ['Full Stack Dev', '2015'],
+    paragraph: 'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    mobileImage: 'images/card3-image.png',
+    desktopImage: 'images/card3.png',
+    list2: ['html', 'css', 'javaScript'],
+    cardButton: 'See Project',
+    cardButton1: 'https://loltolo-lesapiti.github.io/',
+    cardButton2: 'https://github.com/Loltolo-Lesapiti/loltolo-lesapiti.github.io',
+  },
+  {
+    class: 'flex reverse',
+    header2: 'Uber Navigation',
+    heading3: 'Uber',
+    list1: ['Full Stack Dev', '2015'],
+    paragraph: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    mobileImage: 'images/card4-image.png',
+    desktopImage: 'images/card4.png',
+    list2: ['html', 'css', 'javaScript'],
+    cardButton: 'See Project',
+    cardButton1: 'https://loltolo-lesapiti.github.io/',
+    cardButton2: 'https://github.com/Loltolo-Lesapiti/loltolo-lesapiti.github.io',
+  },
+];
+
+// Creating a card div
+cards.forEach((card) => {
+  const cardDiv = document.createElement('div');
+  cardDiv.className = card.class;
+  document.querySelector('.work').appendChild(cardDiv);
+  const imageDiv = document.createElement('div');
+  imageDiv.className = 'flex_image';
+  cardDiv.appendChild(imageDiv);
+  const ContentDiv = document.createElement('div');
+  ContentDiv.className = 'flex_content';
+  cardDiv.appendChild(ContentDiv);
+  // The title header
+  const heading2 = document.createElement('h2');
+  ContentDiv.appendChild(heading2);
+  heading2.textContent = card.header2;
+
+  const canopyDiv = document.createElement('div');
+  canopyDiv.className = 'canopy formodal';
+  ContentDiv.appendChild(canopyDiv);
+  // Images for mobile and desktop
+  const mobileImage = document.createElement('img');
+  mobileImage.className = 'mobile_image';
+  mobileImage.src = card.mobileImage;
+  imageDiv.appendChild(mobileImage);
+  const DesktopImage = document.createElement('img');
+  DesktopImage.className = 'desktop_image';
+  DesktopImage.src = card.desktopImage;
+  imageDiv.appendChild(DesktopImage);
```

Comment:
- [ ] It would be better to stick to the Figma design and try to align the image of the pop up window in the center in order to match to Figma design 
![image](https://user-images.githubusercontent.com/61790587/179957986-b8fe1b04-6f28-4ca1-960f-8b6516953652.png)

Reply:
Hello, @youmari I made the changes now. Kindly check.